Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -119,15 +119,9 @@ public bool IgnoreCollidersNotInFOV
 
         private void Awake()
         {
-            queryBuffer = new Collider[sceneQueryBufferSize];
-        }
-
-        protected void OnValidate()
-        {
-            Debug.Assert(line != null, this);
-            Debug.Assert(visuals != null, this);
             touchableDistance = Mathf.Max(touchableDistance, 0);
```

Comment:
I think these two lines are valid in `OnValidate`, since they're checking to ensure specific fields are within certain ranges. From https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnValidate.html:
>You can use this to ensure that when you modify data in an editor, that data stays within a certain range.

Reply:
The other lines fail because they depend on scene objects that might not have loaded into the scene graph yet. These just refer to floats

Reply:
Yep, I called this out in my description, someone read it yay.

I will change it back to keep OnValidate as you say :)

Reply:
>I called this out in my description

Oops 👀